virtual_ecosystem/models/abiotic/microclimate.py

```diff
@@ -197,121 +192,192 @@ def run_microclimate(
         layer_structure.index_filled_atmosphere
     ].to_numpy()
 
-    # -------------------------------------------------------------------------
-    #  Calculate atmospheric background variables
-    # -------------------------------------------------------------------------
+    # Get combined evapotranspiration from plant and hydrology model, per hour
+    evapotranspiration = data["canopy_evaporation"] + data["transpiration"]
+
+    # If hourly input data is provided, iterate over day, else equilibrium assumption
+    hourly_time_interval = int(time_interval / core_constants.seconds_to_hour)
```

Comment:
Not sure if it is a risk, but if `time_interval < core_constants.seconds_to_hour` this result in `hourly_time_interval = 0`. As that probably makes no sense, given that you use this below below in a denominator, I would do:
 ```suggestion
    hourly_time_interval = max(int(time_interval / core_constants.seconds_to_hour), 1)
```